reference/7.1/Microsoft.PowerShell.Core/About/about_Operators.md

```diff
@@ -337,10 +337,18 @@ Converts or limits objects to the specified type. If the objects cannot be
 converted, PowerShell generates an error.
 
 ```powershell
-[DateTime]"2/20/88" - [DateTime]"1/20/88"
-[Int] (7/2)
-[String] 1 + 0
-[Int] '1' + 0
+# Dates
+[DateTime] '2/20/88' - [DateTime] '1/20/88' -EQ [TimeSpan] '31'
+# Bankers’ rounding
+[Int] (7/2) -EQ 4 -AND [Int] (9/2) -EQ 4
+# Concatenation
+[String] 1 + 0 -EQ '10'
+# Addition
+[Int] '1' + 0 -EQ 1
+# Hexadecimal
+[Int] '0XF' -EQ 0XF -AND [Int] '#F' -EQ 0XF -AND [Int] '&HF' -EQ 0XF
+# Binary
+[Int] '0B1111' -EQ 0B1111
```

Comment:
Hex and binary numbers are covered, in detail, in about_Numeric_Literals.

Reply:
Observe that `#F` is not a numeric literal but `[INT] '#F'` is valid.

Reply:
Yes, but this documentation is not about all the different ways the type-casting can convert values. This is about the existence of the `[type]` cast operator. These more explicit examples belong elsewhere.

Reply:
I gave these specific examples as a hint that conversions are weird (sometimes) and the coder should be prepared to handle unexpected results.  I have removed the examples at your request.